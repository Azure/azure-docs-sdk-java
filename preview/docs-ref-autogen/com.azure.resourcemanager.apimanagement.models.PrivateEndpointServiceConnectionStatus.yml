### YamlMime:JavaType
uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus"
fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus"
name: "PrivateEndpointServiceConnectionStatus"
nameWithType: "PrivateEndpointServiceConnectionStatus"
summary: "Defines values for PrivateEndpointServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointServiceConnectionStatus extends ExpandableStringEnum<PrivateEndpointServiceConnectionStatus>"
constructors:
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  name: "PrivateEndpointServiceConnectionStatus()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PrivateEndpointServiceConnectionStatus()"
  syntax: "public PrivateEndpointServiceConnectionStatus()"
fields:
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.PENDING"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "PrivateEndpointServiceConnectionStatus.PENDING"
  summary: "Static value Pending for PrivateEndpointServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus PENDING"
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.REJECTED"
  summary: "Static value Rejected for PrivateEndpointServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateEndpointServiceConnectionStatus REJECTED"
methods:
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "PrivateEndpointServiceConnectionStatus.fromString(String name)"
  summary: "Creates or finds a PrivateEndpointServiceConnectionStatus from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static PrivateEndpointServiceConnectionStatus fromString(String name)"
  returns:
    description: "the corresponding PrivateEndpointServiceConnectionStatus."
    type: "<xref href=\"com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.values()"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.values()"
  name: "values()"
  nameWithType: "PrivateEndpointServiceConnectionStatus.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<PrivateEndpointServiceConnectionStatus> values()"
  returns:
    description: "known PrivateEndpointServiceConnectionStatus values."
    type: "<xref href=\"java.util.Collection?alt=java.util.Collection&text=Collection\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus?alt=com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus&text=PrivateEndpointServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.apimanagement.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-apimanagement:1.0.0-beta.3
