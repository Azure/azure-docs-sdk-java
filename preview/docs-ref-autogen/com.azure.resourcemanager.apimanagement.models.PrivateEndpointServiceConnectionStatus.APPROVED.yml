### YamlMime:JavaMember
uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.apimanagement.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.apimanagement.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-apimanagement:1.0.0-beta.3
